Add Suricata

satta opened the pull request:
This project lints correctly:
```
$ npx awesome-lint
✔ Linting
$
```
and has been out there for 30 days. Let's get it into the list! :)

<!-- Congrats on creating an Awesome list! 🎉 -->

<!-- Please fill in the below placeholders -->

**https://github.com/satta/awesome-suricata#readme**

This list contains resources in support of Suricata, a Free intrusion detection/prevention system and Network Security Monitoring engine. Suricata is a state of the art software tool that is used by many organizations, and that has spawned an active community of users and developers providing software, information and rulesets to support Suricata's mission.

As a member of that community, I compiled this list to gather these resources and create some visibility for the awesome projects out there!

My 2 reviews:

* https://github.com/sindresorhus/awesome/pull/2391#issuecomment-1352326001
* https://github.com/sindresorhus/awesome/pull/2442#issuecomment-1352336421

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
- You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- Your entry should be added at the bottom of the appropriate category.
- The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- Non-generated Markdown file in a GitHub repo.
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate. Please search for existing submissions.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

satta commented:
`unicorn`

sindresorhus commented:
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

satta commented:
Thanks! Awaiting reviews. 👍🏻 

> I noticed multiple things that are not followed

Huh. I can't find anything obvious, even after going through the list multiple times. Just to be safe, I did the following changes:

* Renamed `LICENSE.md` to `LICENSE` and changed to plain text. I wouldn't think that would change anything about the readability and information content, but if it's that important...
* Changed the wording in the description, by moving the text describing Suricata into the non-quoted text and using the quoted text to describe the list, as the other lists do.
* Removed the line mentioning that contributions are welcome.
* Note that there is no logo because I don't want to use possibly copyrighted project logos, and it says "have a logo wherever possible" which does not sound like a hard requirement.

If there's anything else, just let me know.

satta commented:
After reaching out to the OISF/Suricata team and confirming that I am permitted to use it, provided I do not alter the logo in any way, I now also added a logo including a link back to the Suricata home page https://suricata.io.

satta commented:
Just resolved the merge conflict that came up in the meantime.

I am curious: Any other feedback so far? Is there anything else I can do to move this forward?

satta commented:
Hello? Does anyone actually check any updates here?

From what I can see, I have addressed all points in the requirements that might be debatable even in the slightest sense. If there is anything left for me to do, please let me know. A one-word pointer would do, no need for any detailed explanation.

abdansyakuuro commented:
> This project lints correctly:
> ```
> $ npx awesome-lint
> ✔ Linting
> $
> ```
> and has been out there for 30 days. Let's get it into the list! :)
> 
> <!-- Congrats on creating an Awesome list! 🎉 -->
> 
> <!-- Please fill in the below placeholders -->
> 
> **https://github.com/satta/awesome-suricata#readme**
> 
> This list contains resources in support of Suricata, a Free intrusion detection/prevention system and Network Security Monitoring engine. Suricata is a state of the art software tool that is used by many organizations, and that has spawned an active community of users and developers providing software, information and rulesets to support Suricata's mission.
> 
> As a member of that community, I compiled this list to gather these resources and create some visibility for the awesome projects out there!
> 
> My 2 reviews:
> 
> * https://github.com/sindresorhus/awesome/pull/2391#issuecomment-1352326001
> * https://github.com/sindresorhus/awesome/pull/2442#issuecomment-1352336421
> 
> ### By submitting this pull request I confirm I've read and complied with the below requirements 🖖
> 
> **Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**
> 
> ## Requirements for your pull request
> 
> - **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
> - **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**
> 	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.
> - You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
> - This pull request has a title in the format `Add Name of List`.
> 	- ✅ `Add Swift`
> 	- ✅ `Add Software Architecture`
> 	- ❌ `Update readme.md`
> 	- ❌ `Add Awesome Swift`
> 	- ❌ `Add swift`
> 	- ❌ `add Swift`
> 	- ❌ `Adding Swift`
> 	- ❌ `Added Swift`
> - Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
> 	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
> 	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
> 	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
> 	- ❌ `- [Framer](…)`
> 	- ❌ `- [Framer](…) - prototyping interactive UI designs`
> - Your entry should be added at the bottom of the appropriate category.
> - The title of your entry should be title-cased and the URL to your list should end in `#readme`.
> 	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
> - The suggested Awesome list complies with the below requirements.
> 
> ## Requirements for your Awesome list
> 
> - **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
> - Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
> - Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
> - The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
> - **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
> 	- ✅ `Mobile operating system for Apple phones and tablets.`
> 	- ✅ `Prototyping interactive UI designs.`
> 	- ❌ `Resources and tools for iOS development.`
> 	- ❌ `Awesome Framer packages and tools.`
> - It's the result of hard work and the best I could possibly produce.
> 	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
> - The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
> 	- ✅ `awesome-swift`
> 	- ✅ `awesome-web-typography`
> 	- ❌ `awesome-Swift`
> 	- ❌ `AwesomeWebTypography`
> - The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
> 	- ✅ `# Awesome Swift`
> 	- ✅ `# Awesome Web Typography`
> 	- ❌ `# awesome-swift`
> 	- ❌ `# AwesomeSwift`
> - Non-generated Markdown file in a GitHub repo.
> - The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
> - Not a duplicate. Please search for existing submissions.
> - Only has awesome items. Awesome lists are curations of the best, not everything.
> - Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
> - Includes a project logo/illustration whenever possible.
> 	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
> 	- The image should link to the project website or any relevant website.
> 	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
> - Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
> - Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
> 	- Should be placed on the right side of the readme heading.
> 		- Can be placed centered if the list has a centered graphics header.
> 	- Should link back to this list.
> - Has a Table of Contents section.
> 	- Should be named `Contents`, not `Table of Contents`.
> 	- Should be the first section in the list.
> 	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
> 	- Must not feature `Contributing` or `Footnotes` sections.
> - Has an appropriate license.
> 	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
> 		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
> 	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
> 	- Place a file named `license` or `LICENSE` in the repo root with the license text.
> 	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
> 	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
> - Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
> 	- The file should be named `contributing.md`. Casing is up to you.
> 	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
> 	- The section should not appear in the Table of Contents.
> - All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
> - Has consistent formatting and proper spelling/grammar.
> 	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
> 	- The description starts with an uppercase character and ends with a period.
> 	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
> - Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
> - Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
> - Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.
> 
> **Go to the top and read it again.**
>

satta commented:
Not sure what the above comment is supposed to tell me... 🤔

claire-lex reviewed (commented):
The list seem to follow the requirements.
It would be good to explicitly state the words IDS and IPS, as acronyms can be confusing.

satta commented:
> The list seems to follow the requirements. It would be good to explicitly state the words IDS and IPS, as acronyms can be confusing.

Thanks a lot for the review, I was already thinking this was stalled :)
Sure, I will make sure to explain these acronyms.

sindresorhus commented on readme.md:
```suggestion
- [Suricata](https://github.com/satta/awesome-suricata#readme) - Intrusion detection/prevention system and network security monitoring engine.
```

sindresorhus commented:
Linkify `Suricata ` in `Suricata is a free intrusion detection/prevention system (IDS/IPS) and network security monitoring engine.` to its website.

satta commented:
> Linkify `Suricata ` in `Suricata is a free intrusion detection/prevention system (IDS/IPS) and network security monitoring engine.` to its website.

Done.

sindresorhus commented:
Tweet: https://twitter.com/awesome__re/status/1671458477661736960

satta commented:
Thanks a lot for making this happen!
